client/src/components/DataFiles/DataFilesTable/DataFilesTable.scss

```diff
@@ -8,7 +8,7 @@
 .tr {
   display: flex;
   align-items: center;
-  font-size: 16px;
+  font-size: 12px;
```

Comment:
Independently, I am certain, because it matches the design.

As a team, I am not certain, because your point is valid.

I'll ask Design. Task added to PR desc.

<details>
<summary>Design</summary>

Matt and I know the fonts are a bit small in the interface design. He has conceded that the fonts should be 1.2 times the size of those in the design. After the font is at least 12px, then I don't care the size, because user can change font size (albeit with caveats I don't go into—see "Intent" for solution to those unmentioned caveats).

</details>

<details>
<summary>Notes</summary>

The `12.48px` is definitely wrong. The `16px` is probably wrong. I held back changing font sizes, but then I needed to handle column widths. Handling column widths when font size changes is effort that would need to be re-done once font management is "made better" (see "Intent"). So, I have changed the font size for tables.

</details>

<details>
<summary>Intent</summary>

__After__ all the fonts match the design, __then__ I can _easily_ convert the font sizes to `rem` and/or use variables for font sizes (converting as I got is hard because the font sizes are inconsistent). __And then__, user and/or dev can scale fonts easily and proportionally. I have been working toward that off and on since I joined (there are many comments in the code).

</details>

Reply:
Thanks for getting clarification. IMO, `12px` is too small, especially for older users. I have heard a `14px` is a good rem, and even `16px` for body content

Reply:
Yes, I agree. I am not old yet, but the font size bothers me. Correct, 12px is a non-ideal minimum for body content.

I seek a (1) source of truth and (b) solution to easily change font size across the board.

I have [asked Design to review the situation](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1621370789000200). I will also propose solutions publicly, and schedule a private meeting with Design.

Reply:
Design is firm about the decision to have the fonts match the design (×1.2). I chatted with them privately early yesterday evening. I looked for material to support an opposing viewpoint, but could not build a strong case. [I posted the decision this morning (and my research).](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1621437099007000)

<details>
<summary>N.B.</summary>

Though this change to table fonts was a larger step than I had previously taken to update font sizes, many interface areas (including some tables) already had the Design-expected font sizes.

</details>

<details>
<summary>P.S.</summary>

There are still more font sizes to change, but I believe that can be done piecemeal (not in this PR) like the many smaller updates before this.

</details>

Reply:
Shall I revert the _table_ font size change in _this_ PR?

It can be argued as out of scope. It was done to reduce cropping of table content on small screens (but supported by the fact that the fonts did not match the design).

Reply:
Table font sizes updated to 14px (and table column widths updated to adapt to that change) in https://github.com/TACC/Core-Portal/pull/401 (merged).

Reply:
I have not yet updated the PR with all the font size changes (and related changes) i.e. new "Photos", new "Testing", new "Changes", etcetera. But it is ready for testing.

__Known Issues__

4. `<SiteSearch>` not using `<Section*>` components.
5. `<OnboardingAdmin>` not using `<Section*>` components.
6. `<DataFiles*Listing>` column widths aren't as "pretty" as other tables (because these are not real tables).